domain/src/main/java/com/teamagam/gimelgimel/domain/utils/PreferencesUtils.java

```diff
@@ -32,6 +32,10 @@ public void toggleCoordinateSystemPrefs() {
     mUserPreferencesRepository.setPreference(USE_UTM_PREF_KEY, !shouldUseUtm());
   }
 
+  public void setUsername(String username) {
```

Comment:
Nice.
The `SetUsernameAlertDialogBuilder` should use this too.
(The current impl. use prefs directly I think)